packages/tc-markdown-parser/lib/parse-multiline-definition.js

```diff
@@ -17,6 +17,10 @@ const createNode = (key, value, line, column) =>
 		key,
 		position: {
 			start: {
+				line,
+				column: 0,
+			},
+			end: {
```

Comment:
the value for column is taken when the parser _finishes_ parsing a nested property, so I think it was a bug before to use it as `start`